### PR TITLE
Add per account spreads

### PR DESCRIPTION
**Pull request by gakonst**
Fixes #301

### CODE REVIEW

File: crates/interledger-store/src/redis/lua/load_accounts.lua [hunk 1]
```diff
@@ -38,6 +39,17 @@ for index, id in ipairs(ARGV) do
             end
         end
 
+        -- If the account does not have a spread specified
+        -- but there is one configured for that currency, set the
+        -- account to use that url
```
Comment (gakonst) [inline crates/interledger-store/src/redis/lua/load_accounts.lua]:
```suggestion
        -- account to use that spread
```

File: crates/interledger-store/src/redis/mod.rs [hunk 2]
```diff
@@ -1279,6 +1280,36 @@ impl NodeStore for RedisStore {
                 }),
         )
     }
+
+    fn set_spreads(
+        &self,
+        asset_to_spread_map: impl IntoIterator<Item = (String, f64)>,
+    ) -> Box<dyn Future<Item = (), Error = ()> + Send> {
+        let asset_to_spread_map: Vec<(String, f64)> = asset_to_spread_map.into_iter().collect();
+        debug!("Setting settlement engines to {:?}", asset_to_spread_map);
```
Comment (gakonst) [inline crates/interledger-store/src/redis/mod.rs]:
```suggestion
        debug!("Setting per token spreads to {:?}", asset_to_spread_map);
```

**Comment by emschwartz** [comment]
As mentioned in https://github.com/interledger-rs/interledger-rs/issues/301#issuecomment-565751200, I'm not convinced this is a feature we should include now

**Comment by gakonst** [comment]
Closing since this is stale

**Pull request closed without merging**